recipes/napari-yolov5/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "napari-yolov5" %}
+{% set version = "0.2.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-yolov5-{{ version }}.tar.gz
+  sha256: ec13e9a6c3e0148d77a9f5b4b08798a6b82f190b280a29316da770df195a7ec1
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - connected-components-3d >=3.6.0
+    - matplotlib-base >=3.2.2
+    - napari
+    - napari-plugin-engine >=0.1.4
+    - numpy >=1.18.5
+    - opencv >=4.1.2
+    - pillow >=7.1.2
+    - python >=3.7
+    - pyyaml >=5.3.1
+    - requests >=2.23.0
+    - scipy >=1.4.1
+    - seaborn >=0.11.2
+    - tensorboard >=1.15.0
+    - pytorch >=1.9.0
+    - torchvision >=0.8.1
+    - tqdm >=4.41.0
+
+test:
+  imports:
+    - napari_yolov5
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/rdemets/napari-yolov5
+  summary: Plugin adapted from Ultralytics to bring YOLOv5 into Napari
+  license: GPL-3.0-only
```

Comment:
@rdemets, one question: is this plugin vendoring (copying over) most of `YOLOv5`'s source code? If that's the case, you should specify this clearly in your own license (which needs to be GPL3, that's correct now), and _then_ distribute YOLO's license too. Let me know if you have questions!

Reply:
Hi @jaimergp, it is indeed mostly coming from Ultralytics YOLOv5 repositories. I picked the license to be GPL-3.0 since it is also the license that they are using, and that´s literally the first line of my readme. I am not sure to understand what you mean about distributing YOLO´s license as well. Sorry about that. 

Regards, Richard

Reply:
Thanks for the quick answer @rdemets! The original work is covered by GPL3, which requires we distribute it along with the code. On top of that, you added some more code, also covered by GPL3, right? I am not sure if we need to add two copies (with potentially different copyright?) of the GPL3 license (the original YOLO plus yours), or just one.

@conda-forge/core @conda-forge/staged-recipes, any advice on the legal aspects discussed in this minithread? Thanks!

Reply:
I think it can be the same license file but then it should include both copyright on top?

Reply:
What @jaimergp is trying to say is that if Ultralytic's code is included in its entire unmodified form, then we would need two licenses because then you would just be redistributing the original. Their original license is for their code and your code is under your license. ~~Though if this was the case, I don't know why you wouldn't just list their package as a dependency and import the desired components.~~ It looks like [yolov5](https://github.com/ultralytics/yolov5) is just a collection of python scripts and files, it isn't actually pip installable.

However, it sounds like only *some* of the code from Ultralytics was copied and others was replaced, so you have a modified version of their project. In that case, you only need one license (your own GPL3 license), but in your notice you need to more clearly specify which version of Ultralytics code you modified (there are different 8 releases now), and what you changed so people do not confuse the two or expect them to behave the same. I would expect a more detailed notice like "This project was adapted from ultralytics/yolov5 release 6.1; the model architectures are the same, but packaged as a proper installable python module with entry-points for Napari."

Reply:
Hi @rdemets, did you have time to look into this? If you need help, let me know!

Reply:
Hi, I've modified my GPL3 license file to try to make it more explicit and added the sentence @carterbox suggested. If there is anything I missed, please let me know.

Reply:
Excellent! You'll need to a cut a new release on PyPI so we can get the latest source in a robust way, then! 0.2.12 would be enough!

Reply:
Hi @jaimergp,

I am jumping back on this. I just want to make sure I am not missing anything when I reupload.
Can I just confirm that I need to reupload the folder that I have on my github with the new GPL3 license and the recipes folder that we edited here ? 
Sorry if the question is silly. It's a first time for me. 

Thanks.
Regards, Richard.

Reply:
I see that you have updated the license in your GitHub repo; good.

1. Make a new release on PYPI with the new license. https://pypi.org/project/napari-yolov5/#history
2. Update the recipe file here with the new version number; version 0.2.12.

Reply:
Hi @carterbox,

I did make the pypi new release 0.2.12, but I can't change the meta.yaml file :

``
This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.
``

Reply:
You are probably pushing to the wrong remote or you don't have permission to edit goanpeca's fork.